lib/lita/handlers/jira_issues.rb

```diff
@@ -80,6 +82,26 @@ def priority(data)
       def issue_link(key)
         "\n#{config.url}/browse/#{key}"
       end
+
+      def silenced?(key)
+
+        if config.issue_ttl == 0
+          log.debug("JiraIssues: issue_ttl is set to 0, will post every matched issue to chat")
```

Comment:
Are you finding the debug statements particularly useful? In some sense you've verified the behaviour in tests (:+1:), so this is kind of extra. Also, without the log statements most of this code could be cleaned up very nicely.